### PR TITLE
Claims, Flags, Peggy IBC Whitelist

### PR DESCRIPTION
**Pull request by mccallofthewild**

### CODE REVIEW

File: ui/app/src/views/RewardsPage/components/RewardSection.tsx [hunk 1]
```diff
@@ -157,16 +157,27 @@ export const RewardSection = defineComponent({
                 Learn More
               </Button.Inline>{" "}
               <Button.Inline
-                onClick={() => props.onClaimIntent()}
+                onClick={() => {
+                  if (
+                    !import.meta.env.PROD &&
```
Comment (ajoslin) [inline ui/app/src/views/RewardsPage/components/RewardSection.tsx:162]: import.meta.env.PROD means nothing, I think it's always true if things are minified

Do VITE_APP_DEPLOYMENT!=='production'

Comment (ajoslin) [reply ui/app/src/views/RewardsPage/components/RewardSection.tsx:162]: although might not want to do that..

this condition will just always show alert in local dev env which is much safer

We don't want to risk a prod deployment where we don't set VITE_APP_DEPLOYMENT correctly causing this to show up

Comment (mccallofthewild) [reply ui/app/src/views/RewardsPage/components/RewardSection.tsx:162]: If prod is always true, this condition will never trigger in any environment. I'll refactor to check if VITE_APP_DEPLOYMENT == test | develop | etc.

Comment (mccallofthewild) [reply ui/app/src/views/RewardsPage/components/RewardSection.tsx:162]: Fixed in https://github.com/Sifchain/sifchain-ui/pull/216

**Pull request merged**